ui/components/app/edit-gas-fee-popover/network-status/status-slider/status-slider.js

```diff
@@ -14,8 +16,15 @@ const GRADIENT_COLORS = [
 ];
 
 const StatusSlider = () => {
+  // todo: value below to be replaced with dynamic values from api once it is available
+  // corresponding test cases also to be added
   const statusValue = 0.5;
   const sliderValueNumeric = Math.round(statusValue * 10);
+
+  let statusLabel = 'stable';
+  if (statusValue <= 0.33) statusLabel = 'notBusy';
```

Comment:
Do you mind putting these if statements on their own line? I'm not sure why ESLint isn't picking this up, but I think it'll make this logic more obvious.

Reply:
fixed 👍